ubuntu-1804-image_5b461f730428630abc0bf442.md

```diff
@@ -15,14 +15,16 @@
 * [Ruby](#ruby)
 * [See also](#see-also)
 
-
 ## Overview
 
 The `ubuntu1804` image is based on [Ubuntu 18.04 LTS](https://wiki.ubuntu.com/BionicBeaver/ReleaseNotes).
 The user in the environment is named `semaphore`, and has full `sudo` access.
 
 Image name: `ubuntu1804`
 
+Currently the `ubuntu1804` Virtual Machine uses an *APT mirror*, which means that
+caching packages will have little effect.
```

Comment:
The `ubuntu1804` Virtual Machine uses an *APT mirror* which is in the same data center as our build cluster, which means that caching packages will have little effect.